library/src/morpheus-master/js/src/sakai.morpheus.more.sites.js

```diff
@@ -421,11 +455,11 @@ $PBJQ(document).ready(function($){
     $PBJQ(btn).data('favorite-state', state);
 
     if (state === 'favorite') {
-      $PBJQ(btn).attr('title', $PBJQ('#removeFromFavoritesText').text().replace("[site]", $PBJQ(btn).parent().find('span.fullTitle').text() ));
+      $PBJQ(btn).attr('aria-pressed', true);
```

Comment:
Used [aria-pressed](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed) to let the SR know if the star button is pressed or not.